Specs/DateTime/German/DateTimeModel.json

```diff
@@ -145,17 +175,17 @@
     ]
   },
   {
-    "Input": "Ich komme dann Freitag zurück",
+    "Input": "Ich komme am Freitag zurück",
     "Context": {
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
-    "NotSupported": "dotNet, javascript, python",
+    "NotSupported": "javascript, python, dotNet",
     "Results": [
       {
-        "Text": "Freitag",
-        "Start": 0,
-        "End": 0,
-        "TypeName": "datetimeV2.date",
+        "Text": "am freitag ",
+        "Start": 10,
+        "End": 20,
+        "TypeName": "datetimeV2.set",
```

Comment:
Should "am Freitag" really be a set? Set means a recurrent date, like "every monday".